Projects/DesignSystem/Sources/UIComponents/WKInputTextField.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  WKInputTextField.swift
+//  DesignSystem
+//
+//  Created by madilyn on 2022/11/17.
+//  Copyright © 2022 com.workit. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+import RxSwift
+import RxCocoa
+
+public class WKInputTextField: UITextField {
+    
+    // MARK: Components
+    private let clearButton: UIButton = UIButton()
+    
+    // MARK: Properties
+    private let disposeBag: DisposeBag = DisposeBag()
+    
+    // MARK: Initialization
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.setDefaultLayout()
+        self.setDefaultStyle()
+        self.setTextFieldClearBtn(textField: self, targetClearButton: self.clearButton)
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+    
+    // MARK: Functions
+    /// textField-btn 에 clear 기능 세팅하는 함수
+    private func setTextFieldClearBtn(textField: UITextField, targetClearButton: UIButton) {
+        textField.rx.text
+            .orEmpty
+            .distinctUntilChanged()
+            .subscribe(onNext: { changedText in
+                targetClearButton.isHidden = changedText != "" ? false : true
```

Comment:
```suggestion
                targetClearButton.isHidden = changedText.isEmpty
```
요렇게 바꿔줄 수 있을 것 같아요 ~!